docs/integration.rst

```diff
@@ -1,12 +1,72 @@
 Integration
 ===========
 
+- :ref:`ReverseProxy`
 - :ref:`Azure`
 - :ref:`AWS`
 - :ref:`Databricks`
 - :ref:`GCP`
 
 
+.. _ReverseProxy:
+Reverse Proxy
+-------------
+
+Airflow can be set up behind a reverse proxy, with the ability to set its endpoint with great
+flexibility.
+
+For example, you can configure your reverse proxy to get:
+
+::
+
+    https://lab.mycompany.com/myorg/airflow/
+
+To do so, you need to set the following setting in your `airflow.cfg`::
+
+    web_server_url_prefix = /myorg/airflow
+
+Additionally if you use Celery Executor, you can get Flower in `/myorg/flower` with::
+
+    flower_url_prefix = /myorg/flower
+
+Your reverse proxy (ex: nginx) should be configured as follow:
+
+- pass the url and http header as it for the Airflow webserver, without any rewrite, for example::
+
+      server {
+        listen 80;
+        server_name lab.mycompany.com;
+
+        location /myorg/airflow/ {
+            proxy_pass http://localhost:8080;
+            proxy_set_header Host $host;
+            proxy_redirect off;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection "upgrade";
+        }
+      }
+
+- rewrite the url for the flower endpoint::
+
+      server {
+          listen 80;
+          server_name lab.mycompany.com;
+
+          location /myorg/flower/ {
+              rewrite ^/myorg/flower/(.*)$ /$1 break;  # remove prefix from http header
+              proxy_pass http://localhost:5555;
+              proxy_set_header Host $host;
+              proxy_redirect off;
+              proxy_http_version 1.1;
+              proxy_set_header Upgrade $http_upgrade;
+              proxy_set_header Connection "upgrade";
+          }
+      }
+
+Azure: Microsoft Azure
+----------------------
```

Comment:
I don' think this header addition should be here

Reply:
indeed

Reply:
I forgot to remove it indeed